src/lndk_offers.rs

```diff
@@ -160,8 +162,17 @@ impl OfferHandler {
         let pubkey =
             PublicKey::from_slice(&pubkey_bytes).expect("failed to deserialize public key");
 
+        // Generate a new payment id for this payment.
+        let bytes = self.messenger_utils.get_secure_random_bytes();
+        // We need to add some metadata to the invoice request to help with verification of the invoice
+        // once returned from the offer maker.
```

Comment:
nit: expand this comment to explain what we're doing here - this is for stateless offers right?
Commit message would also be helpful